packages/suite/package.json

```diff
@@ -72,6 +72,7 @@
         "zxcvbn": "^4.4.2"
     },
     "devDependencies": {
+        "@babel/preset-typescript": "^7.16.7",
```

Comment:
`@babel/preset-typescript` (v 7.13.0) is already installed in top level package.json, do we have to have it here explicitly?

Reply:
true, removed